backends/gstreamer/audio_stream_reader.rs

```diff
@@ -30,7 +30,9 @@ impl GStreamerAudioStreamReader {
 
         // XXXManishearth this is only necessary because of an upstream
         // gstreamer bug
```

Comment:
Which? :)

Reply:
Perhaps you remember from when you suggested this: https://github.com/servo/media/pull/362

Reply:
Oh no :sweat_smile: https://github.com/servo/media/pull/362#issuecomment-647947034 has the infos apparently. Maybe this can be added to the code :) Probably unnecessary workaround nowadays.

Reply:
Added a link to the comment. ;)